app/upgrades/v24/upgrades_test.go

```diff
@@ -131,6 +131,10 @@ func (s *UpgradeTestSuite) TestUpgrade() {
 	oldBaseDenoms, err = s.App.ProtoRevKeeper.DeprecatedGetAllBaseDenoms(s.Ctx)
 	s.Require().NoError(err)
 	s.Require().Empty(oldBaseDenoms)
+
+	lockableDurations := s.App.IncentivesKeeper.GetLockableDurations(s.Ctx)
+	s.Require().Equal(len(lockableDurations), 1)
+	s.Require().Equal(lockableDurations[0], time.Hour*24*14)
```

Comment:
Test should check lockable durations before and after, to make sure that the upgrade triggered this and it isn't just what was defaulted to.

Reply:
Added according check in the most recent commit!